WalletWasabi.Fluent/ViewModels/Wallets/Settings/WalletCoinJoinSettingsViewModel.cs

```diff
@@ -113,6 +122,25 @@ public WalletCoinJoinSettingsViewModel(UiContext uiContext, IWalletModel walletM
 					}
 				});
 
+		this.WhenAnyValue(x => x.ExcludeCoinFromCoinjoinThreshold)
+			.Skip(1)
+			.Throttle(TimeSpan.FromMilliseconds(2000))
+			.ObserveOn(RxApp.TaskpoolScheduler)
+			.Subscribe(
+				x =>
+				{
+					if (x == "" && _wallet.Settings.ExcludeCoinFromCoinjoinThreshold.ToUnit(MoneyUnit.BTC) != Constants.DefaultExcludeCoinFromCoinjoinThreshold)
```

Comment:
When removing everything, it means that I want to disable the feature, so I want the value to be 21000000. This is again the cost of not using `null`